src/calculateRentalCost.js

```diff
@@ -1,10 +1,24 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCosts = 40;
```

Comment:
It is a good practice to constants with uppercase and underline like this `const DAY_COST = 40;`